### PR TITLE
Added two new Skaraan specials introduced in v2.7.2

### PR DESCRIPTION
**Pull request by JordanLongstaff**
As mentioned [here](https://artemis.forumchitchat.com/post/artemis-2-7-4-is-now-released-10384883), there are two new elite special abilities for Skaraan ships as of Artemis version 2.7.2.

- TeleBack: Ship can jump backwards
- ShldReset: Ship can scramble its own shield frequencies

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1232,6 +1232,18 @@ <h3>Special Ability</h3>
                   <td>ShldVamp</td>
                   <td>Can steal energy from player ship shields (unconfirmed)</td>
                 </tr>
+                <tr>
+                  <td><code>0x1000</code></td>
+                  <td>ELITE_?</td>
```
Comment (NoseyNick) [inline index.html:1237]: Damn, these aren't documented in mission-file-docs.txt yet? That feels like a bugreport   :confounded:

File: index.html [hunk 2]
```diff
@@ -1232,6 +1232,18 @@ <h3>Special Ability</h3>
...
+                  <td>TeleBack</td>
```
Comment (NoseyNick) [inline index.html:1238]: I assume therefore this is a name you have made up, not "official" in any way?

**Review by NoseyNick — APPROVED** [review]
Provisionally approving. Would be good if we can get more official names for these, but documenting that they exist, and roughly what they do, still has value. Will await other review ( @rjwut ?) and/or merge in a few days if we don't get any other feedback.

File: index.html (hunk 2 above)
Comment (JordanLongstaff) [reply index.html:1238]: The names "TeleBack" and "ShldReset" are what show up in the intel in the Science console.

File: index.html (hunk 1 above)
Comment (JordanLongstaff) [reply index.html:1237]: I concur.

Comment (NoseyNick) [reply index.html:1237]: Bug reported at https://artemis.forumchitchat.com/post/2-7-4-bug-reports-10388698

**Comment by NoseyNick** [comment]
No other approvals, but it's been a few days, so merging. Thanks @JordanLongstaff !

**Pull request merged**